tests/integration/test_fixtures.py

```diff
@@ -36,7 +36,7 @@ def finalizer():
             cassette_name = ('tests.integration.test_fixtures.'  # Module name
                              'TestPyTestParametrizedFixtures.'  # Class name
                              'test_pytest_fixture'  # Test function name
-                             '[https:-httpbin.org-get]'  # Parameter
+                             '[https---httpbin.org-get]'  # Parameter
```

Comment:
Ah, I missed that in that line. Thanks for explaining it to me. =) I tend to agree that this is a better idea, but I would point out that there are likely people already using this to generate cassette names. With that in mind, they will have to re-record some cassettes or will run into problems otherwise. I'll add a strenuous warning about this in https://github.com/sigmavirus24/betamax/blob/master/HISTORY.rst later =)